tests/integration/api_v3/rpc/generators.test.js

```diff
@@ -29,8 +29,23 @@ const {
 
 const wsRpcUrl = `${config.SERVICE_ENDPOINT}/rpc-v3`;
 const getGenerators = async params => request(wsRpcUrl, 'get.generators', params);
+const getDPoSConstants = async () => request(wsRpcUrl, 'get.dpos.constants');
+
+const STATUS = {
+	ACTIVE: 'active',
+	STANDBY: 'standby',
+};
 
 describe('Generators API', () => {
+	let numberActiveDelegates;
+	let numberStandbyDelegates;
+	beforeAll(async () => {
+		const response = await getDPoSConstants();
+		const constants = response.result.data;
+		numberActiveDelegates = constants.numberActiveDelegates;
+		numberStandbyDelegates = constants.numberStandbyDelegates;
```

Comment:
Can we use object destructuring instead?

Reply:
As per offline discussion, this can not be done because of variables being global scoped.